scripts/ssb-patchfoo/install

```diff
@@ -33,11 +34,13 @@ cp "$BASE_DIR/config" ~/.ssb
 # Stop ssb service to process plugin
 sudo systemctl stop ssb
 
-# Disable the git-ssb prerequisite
-# Comment out two lines in patchwork that create a prerequisite for git-ssb
-# but don't seem to serve any purpose. Git-ssb is not available on npm
+# Disable the git-ssb and npm-ssb prerequisite
+# Comment out two lines in patchwork that create a prerequisite for git-ssb and npm-ssb
+# but don't seem to serve any purpose. git-ssb and and npm-ssbis not available on npm
```

Comment:
```suggestion
```

Those lines do serve a purpose, it's not like it used to be where they were deps for no reason.